Twice.dfy(27,21): Error: assertion violation
Execution trace:
    (0,0): anon0
    Twice.dfy(23,12): anon3_Else
Twice.dfy(35,31): Error: possible violation of function precondition
Execution trace:
    (0,0): anon0
    (0,0): anon10_Else
    (0,0): anon11_Then

Dafny program verifier finished with 4 verified, 2 errors
